src/transformers/models/poolformer/configuration_poolformer.py

```diff
@@ -125,3 +130,20 @@ def __init__(
         self.layer_scale_init_value = layer_scale_init_value
         self.initializer_range = initializer_range
         super().__init__(**kwargs)
+
+
+class PoolFormerOnnxConfig(OnnxConfig):
+
+    torch_onnx_minimum_version = version.parse("1.11")
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict(
+            [
+                ("pixel_values", {0: "batch", 1: "num_channels", 2: "height", 3: "width"}),
+            ]
+        )
+
+    @property
+    def atol_for_validation(self) -> float:
+        return 1e-2
```

Comment:
`1e-2` looks low for `atol`. I won't go more than `1e-3`, even if `1e-5` is best.

Reply:
@ChainYo Understood, but if I set it anywhere between ```1e-3``` to ```1e-5```, then output values doesn't match while validating. Any suggestions?

Thank you

Reply:
The outputs match on ```2e-3```

Reply:
![image](https://user-images.githubusercontent.com/51019420/198842884-d8da5fd5-384b-427e-9c48-ad1d9cb61fc1.png)

Reply:
Cool, I think `2e-3` is okay! Let's see how @lewtun feels about that.

Reply:
@lewtun ?

Reply:
Please do not multiple pings. It doesn't help :hugs:

Reply:
Sorry😅

Reply:
Also, could you fix the linting for CI?